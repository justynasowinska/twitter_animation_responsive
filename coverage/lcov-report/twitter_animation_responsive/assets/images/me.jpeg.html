<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for twitter_animation_responsive/assets/images/me.jpeg</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">twitter_animation_responsive/assets/images</a> me.jpeg
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>1/1</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a>
<a name='L49'></a><a href='#L49'>49</a>
<a name='L50'></a><a href='#L50'>50</a>
<a name='L51'></a><a href='#L51'>51</a>
<a name='L52'></a><a href='#L52'>52</a>
<a name='L53'></a><a href='#L53'>53</a>
<a name='L54'></a><a href='#L54'>54</a>
<a name='L55'></a><a href='#L55'>55</a>
<a name='L56'></a><a href='#L56'>56</a>
<a name='L57'></a><a href='#L57'>57</a>
<a name='L58'></a><a href='#L58'>58</a>
<a name='L59'></a><a href='#L59'>59</a>
<a name='L60'></a><a href='#L60'>60</a>
<a name='L61'></a><a href='#L61'>61</a>
<a name='L62'></a><a href='#L62'>62</a>
<a name='L63'></a><a href='#L63'>63</a>
<a name='L64'></a><a href='#L64'>64</a>
<a name='L65'></a><a href='#L65'>65</a>
<a name='L66'></a><a href='#L66'>66</a>
<a name='L67'></a><a href='#L67'>67</a>
<a name='L68'></a><a href='#L68'>68</a>
<a name='L69'></a><a href='#L69'>69</a>
<a name='L70'></a><a href='#L70'>70</a>
<a name='L71'></a><a href='#L71'>71</a>
<a name='L72'></a><a href='#L72'>72</a>
<a name='L73'></a><a href='#L73'>73</a>
<a name='L74'></a><a href='#L74'>74</a>
<a name='L75'></a><a href='#L75'>75</a>
<a name='L76'></a><a href='#L76'>76</a></td><td class="line-coverage quiet"><span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">���� JFIF <<  < <  �� C >>>>




�� C<		

��  � �<" ><<��   ><<<           >	<�� C <>   <> !1AQ"aq�2�#��	3BrRb���$C���%DS���� < ><<<          <> �� * >>>><>><      <>!1"AQq2a�#B���� < > ? �HM���A#�X�8��4��{A[M��״X��A�D�H���&amp;&gt;�0T��,]����`�k�&gt;���C	j��� ��%�?��f��zPOO�
�e�4
mh hm���_x;��Yj�+�����zL�����e_D'�[�8g����x�i������ӢQ%g&amp;7��!�/ l�#A�6�F��x�Q�?�:�&amp;�~eG|�-�B�qcʺT�KYiú�&gt;�'�4\�����]=����"�w����k��z��I� ��M�v)p��Z�c���1���} ��𚴘*�'�q�Tߨ�;�Mvu�I��Q{���]M��V���S�؞�}!mLK8mĞ�P6"�oa�
�	�o���ݴ�-��� ��}��H0R�q�Z�-��|����zť?x�����Qz��=��D]Kmm6����`�6��B�_��TM�$�O��	:A>nA:ŭ�i%'��.m�v�{E����X_�E�f٧J;30�Za��-j�<�s?&gt;90�zrR�õ��R�%)h*�fU��D~;&gt;$�ٶ��,=3����j
�VF��1���>~e�Qu��!Pc]��3�q]_bz�r�&lt;��Bu�뮬���� :<Е�ԓ��D0�(����1i ,d?�� Hڼh�[&gt;%�|��_��`�A(9��$j=�\��T<�?�Š����-��O�;AR��#�mhe��"�˭ ���19�l����w�2�q�����#�#�v2���~&gt;&amp;1j�ʉ�g���
��)g%��ƿ���lz��Q�V-��J�Z�𙑛i/4����:(�2E���I6���w�����gr����5G��̹��}"�ly):��y�I-K��ml�wq�Q�6R,���m �bkO]�iOM�4���n[��-��G�&amp;ݠ�,�,���	��?�Th!�;DU�o�#K������1 ��;�����b�.moq�
�5�;Ch@��YN�xi>� ��\�3�x�+�~��&lt;�J�����JӘ�Ѵ{���i:ik�G�����1��w�=1� a>���-��1r�窵'�7?6��&amp;^�qF�� �hRj��Nm�E�4�	����C�BPҦ�ත6$D��RF�iv�Dg:�k�\�q�U��%�]DtJp1z���)R�%�P7lG����s�0j��M�������FZs6��u�.1�kr�.O�M����d���C��FAT�4�ɦo�;ɩ�T����:^���u5�c%�30ZX�����.��u?�W�'���DU�}(<�� ��(ʠ���*�M`9��
-��X2ɜh�&lt;�?X�Pg��T��I���4�]ja�X��
Ы��j#�'6e�4�)���.�̉�r��t�~�T�dY�gq�����Jf&amp;X>i�
�|.<�\_�%��#���h���8q���I����N�t���	��_���o�����թ�@�N�hd��Qb"��b�A��b� /h� ���#�<�KoE�M��-m�-�k�
� �e���m7��Z��]=!�
t0�	�B{o�G�ys�َ���1*ʿ�(%�^� ���_a�{�$��m�ƿ��&lt;�9�:Nf�ϡg�+O��b�¿$��NQ��W��u�?���e	�BR�������� ��9�������84��4�7$~i����@#���%����-Jrdw#��(-��h��Co��Ajx9�W�B�����b+Y�3��\�1�\� (�ް�U��4L�irᱡ(1�ص�QGk�E1�M���	�d��-���+AU#��m��3w�3�4��:.�t��E�XoS�S�,L>&lt;*)&gt;F$5J]I�?�4�38�����L����%�é����+��Лy_���o}:�[H^�������e�é&lt;��p&gt;�tP���U���ʾ���Ф�'@�#C�K�UME�D�ً53+��t�.mӷ��lgn��9ӫn1g*M��L�n�G��a���,T$�Rm��B���:�\=g��|�Z�y�imi)RRA���G&gt;W���	#0w2H����:�Z�G'�/��_�\b0�i�)a�j�"�;fX���x����([q%�,P��F�I(y��{xҠF��=���2�����·e�YK�hQ$�*��Qk���̳��>)���xX�xtP,Q�-#�Z��DM��+�TI��B ��"��$y�	�>�X>=�����M���)���t��J,D<����SkAH��E���2~2x7Q�|L֥�\���jU�4���S�-�ߘ�z�==JrZ�_�qǦ"fO��&gt;�#�.��Ԭ�ʀ�.�)O�q)7�0�r�[��F:���l�-�r��gP���rՙU�PR��)�\ ]JI
����&lt;�Y��ԥ32�.2�/R���bJ��Cav_@ijR��E)#�Y�p�!�ny����T��&amp;]�������*��qe�e�k���īQIi'��j,D*���5���dRR\Nk�@t'K���M&gt;�4�K�M�0fY
���Z�e�k��Z������]g3O��V��ZT��P��-���%lo�H���ؗii�×���)�;�3S��%��T&amp;�<
')Rm��q��C��������=��d����Z��
|@X����!�%���[MX��VW�
��	O�c�΃���|L����p�+�h���J��!��Z��#2�5&lt;��zE9??j&gt;� ?�^.*��oi� ���&lt;N��Uu���BZ�e�����$
}�x�+?��(�S�I�g;�L�K��$$8�Rn,t��}o�m�sN��*�I�EJU�̻+
t#�16���yl�P��`YB�>�n��!���8�,+yB]�ZqG�fG���%̼�k��u�8�6�, ���cr4k�ęiuz�)NViii���y�u=��zW�jcS��9��@T��"�&lt;� �%�I+�;�	�Ǵy��)yv����&amp; aUGT�������2)�pUGO[g�iU\���K����DĚR�/o#��?8�3Fu�Z*3�.�&amp;�|����x[��
��ߨV�e�\y䶔�Q6>:��'06-��@zi	mHO��B첒;���I����sR�隓��H<N��Yk��a�E�SiR��D>8��x&amp;A%��aI�KC+E�CKC.�.��h�"�	1P4�CПϴ(����&gt;��I��H 
�5�a�S��X����6�t�Q��II@ZN��������I+A|��_�S*�*2S ����`��94�k��O�G�h@6״pW����Y*�ѝ�P���{�U{� ���m&amp;�W�k.b%c�m>�ڒ��=&amp;Vm
84��H���r#{?"��jB\C���
H��"��xW��ԩ~�;(�SM0|I"�] �U��c���4�+Ò5js��)�fmde$\�GCpt�?����/�=�\r#���F��:"�sH�*'R%�?Ai����I�bQ��a�!���#y�̶�9�AQ9RS�]OMQ
�+�@GH�&amp;޷��u�ä�)�(�H~]��
M����d䚚�M���K�6s6��9k�[b��\�RS.՛[��� �(��fd����`|�Y�C��'���-�~
��Q_Ѫ�x.��V�&amp;�Q.�*C��L{��cl��a��Hq�T]I�5Y�?'6��^��a��5��n����Қ�����.������l�Up��F�NRi2���,���h�����Q*V���&lt;���s��N��5_4C������Ǧ�\P��\,}ʋ�.TIa�t�����XjN�&lt;�SC�Tw��zE���U��5�4a~�^�Gc� �el�������PڨqF�%1�2�3ͤ�>��{h�}|��'6�Red����[?���Ki�.*��8/��,��%RSSpK^.�eJWoc�(����_p��&gt;�J��˲�oM2��BFd*�$Cx뿔�5-�u�X@�6����� ROAfBe��b���
-��a�	���Pu��:X�Cr!Bo�����������,!PK�-�a&gt;��0����d�����Ќ�\Xu���:�.mm�Դ���BF�ۤh�6�ݬo�']}M3%'�'u�P�r�^.���&gt;�7�Mh"5�^I�k>U0����&gt;R��!���
&nbsp;
)魭�ư�;�E��R��B�T�ؿ�*Y�	���7�}I�;GM�!b�h�-6�ی:��,��l�:��*�� �����g���n�(�ZRZ�C@ʧ.]�y��J;�*=��"k���v����T�P���Kȸ���T�M�&gt;�e�w�ףv5�ǱZA���ҁХ')	���KĬ�&amp;+T/��\KL$)6��V�#��U6�O6
ԅ��ۺI�� �q;�Dy����=�5$�4����Z�����c���:_�H�&gt;sVf�,4�vH!�(���ZQչ5�(��6��q�BR���|��#�E�0����UF$\s���&lt;���m��QmBɹ7��6^g�de�2<�CyV|����Fw&gt;ݣ��S�'V��%.T�ÆQ*M�]�r\u��G2m�$�&amp;[lQNo��|Vcq����x.���-��G*� �
�$!'�"M�
�t��z�T�"�D�I�u)zVi�g�deU����q&gt;����Zq�UqEU�r�0uVut$�}o�I8R���.*�7D�Z�.�R��͓u��͋�t��F�k�!�La��7Z��V��Z�)����s2�4�=�y��u6���r&amp;M�9��h��vZ�J2��f!�7*器 $�):ڛQ"ٳ_�_V�m3�
�Fe,��V%$'֡�k�!s�**��	���n,T0?��!�i*�B�����
���5��S�b���6Ĭ��#;�x��Q0U��l��Ү8Ӌ/0�Zύ=ң�������aRo�	i�ԛ��>[~_��(�H�*�,��T2�~/?8i�g#��4����j��!sM�C
7�Z�/
�ַ��2�Z�u�쫘�[Cm�{l?c����}a����4t���M Gxu�v�[uĭIiؘ,Í�$ff���JuG�)$�����Rz�|�G��u
��&gt;^e�c��V�񂱡Y����g���)U\���Q�e�:�&amp;:����7?�.�V�׭����N��Hȉk�4Ǐm�h��i$����͇�m�����3�"���ъ�����w���l�  �VK^�%�d��zy��hȝ4&amp;�$�vgq=*Y��K%�4����k  � �(Չz.&amp;�NL+�,��8��$���\Ex����_�l�*f���}�
&gt;�3Z�=��\	PIE�?I�#�9W����B�M���#S�ZT�Z�s���u'"�l:[M�wRl�Ҵ���\��:bŷ��&lt;�{�)�|�Y(�uJa��� ���1Sy	3(U�j�
r���	�v'����f���x.�Nyž�Z��M��>��l�����J����K��\o�A�HZR����{Ò������d.�]X� ��"�V�ǝ���V�7��N��"[�Z>���h�ǔ@��"�����	h�H::�$�*y�A!�2��HM�>-B Ӵ2�Z��P��q0Հ��d�]�����2A��q;�K��)v���L�XDy8ꄙu8��&amp;���[ �D��5�4������W��C�C'�&lt;���,<��q���siRE�
���DAq��g%�z���G-�!MR]Y���̟c�R���f悏9*�O�'{��x���9��!�k.ڵmG�bʾt�q7��C"�2f���ǉA�9wpQ�pd�TB���9�+�XyD�#�o%���C�z� -�1.�@>E�u"&lt;�읏sg���SAh���	_�}#0�.��1�d#(�㩌�8>��Z/�H�d�4�"m@p4[}b?+(R�Bs%Q��p2�`
��*��="Z��K�J��i\_] ��.2	��ͭ%����LW�
���
K�OS<���$��+t~�Q��w��!�_���M���;,zk���]H$���#��]"�IЎ���Ͷ����NG�Ր��L�C���j�kG1Rx��pܲe٨7:�,R����<���
��g��;�����g�J�g�(⥜�8�<669MŽc�ї^GQ�S+�n��تn� +j�q�KR����l�E�j�(���8�m0oF5M}��)�[ ��,�@��k���PwQ�&gt;�Q>CG���YG[@�oc>�T� tI] s�� ��}���c���4d�
D�Ǎӯ(�@�������p�b�7'�2ɔ3/��.�nIԟ8���2�<al�,H&amp;5�R�)C&lt;��&gt;�JU��9��3i�H$;�J�P����@18rA
N�.��n�L����&amp;�h�iv� A|��ܴ���*��)��JӺN��JM�̻�:�2Zi�ۨpE�&gt;������
4Z�L�烃U'����{D���&amp;p�nr�4��e�d�����c�[�� ������i����4�I��!:Z�[�ZRc�����0�t��nM����l�}Κ��Zf�o^
���fU+Dc��L$'Kcv'�B3��i�^
;(�e�&gt;͑5UK2-�5�[�Y���l��h��1���M��M51r�NK ���H�.��W*Q*^������ǁ ��f^��U�7;�񝔧0�vJ@P��1%����0��"@Ԙ؜&amp;�:�)�p]��I'r���?d����5��
4� ��L�Y�7ԯD��h�Vh�R�I�I#,�9�6�Q�%)ʛ�f;ޝWng��{R�k���܊��E�xM�_�����ɴ��Re�Q�&lt;¤�������*�&gt;д�R<M�m#��y��'���n��媪mS	&gt;�NP�؎���+��kF����H9�m���ɓR��<��E�pR�'0���q�$�����A�F�^"���}Ui������z�q�y�|ˮ�J֥~p�
*�:��)�D&lt;B��t�U.]2ӫ�A?��_�P�d���h��o�K�U��)�n�u�=�~�� ac�Q"�Z��d��籶��ȅv=�X.�ؚ�F�#xb^d5�6]"�|*���hd�H��c'�
�\
1��f%BQY�O�ZR���=�h0���ͭ��QB�ZHRT7w���p�@b&gt;�l}�Y\d�M�S%WJmη��6� 
G�r��]��&lt;��O�T?n��� ��rik�^�avó�
F������'��,�5ʡp�����)Ǜ�i�ֹ)�Q}��6&amp;ü9�&amp;�1�J�� Z,T2�&gt;�C}�l����a�ȱ*&amp;�!u+B��i�(�"��mn>��Xi���I�� HH&amp;�d>s*é���N�y>q+����
��� g��e
��S��k���U����p��5�yTABB��(X٫�o<�Vʟ {��ҥ��J�l��&gt;��{�cY9̻M!�	\�(@ʛ'@ �:A��j�&amp;fmܲ���{G���TQ�̋��g9v]v�6i)kk��'�Ol%$$Fv��a��H�A�!��MLx���].��+r%&lt;6��}nN�.�+�&lt;�~Γ�]��L9,�V�m"�83x�6�G$�� ��tV��m����$�C3EEb��i�E6i݊ZVޑ�
�:�H���?�6T�q |k�����-@ܲ<;���㦟35L��+W&gt;_������&gt;�0�A^WGһhb,��dg���B�XZ	����/�Y��ZBp7OQUɿ�"���6:۴`�g�0���t$FE�2�?x���ʈ*��!W�-�!���T>���"�+N��:@aG��8�����&lt;�%-�.��@�b�_,���iai�BzB�=�(X���]a
k�n
��z~���{UE!�J�l�ə`���$�I�y�G>�]��Ă����p��i��c�k�Ѧ���Z�ZF���,��\�0�3:H���A�!�J���
Du~A�SI��QM����tB��G5bd�E��b`r�iE*���b*'�.��};�MmX�H���5&amp;���w	HH�R�� u$�xk��h:�/8˘�z<��c�
;��	� ��GP(p��sGT��E��S,��),�i��it�JH9��}�_H�3�)���NuY��kl�T3�S��*[�#�)@� sT&gt;�m{
=cf����\	i6ӑ
�'��<��ݮ�˭Ki%׎ �0�c��A��=#�U5Եy�o�"\�c�m�Vlfӵ��h��:������T��Uk�m��S�F%�5Z�
�
��Z�_����#��R�M�[w���')�Р	VN�Ď��~I���k�3-S��<�� Cqo�W4��d�l$���JH�{CK$7p�����iL!j:�⊕�x87h�v=b�F��Ͼ�ø���*.c�d^�l~�QX��Ū	�z��">⮻
⢠.�bZ�{�/�z�8��؂���q�� �w􊊊߂�?V����ӎ#��J���hF�G�M�S��T�:��$ew�**&lt;ϡ��\!*T�H��`��A!Hbv���,m���<�󋌛��	��TTZ^	#�|F�$�EB�rF�ef9K7e�E���}b��q�L�[H�6Rҹ�*^h�	'�A��<���,�l�
]���(=2C�&lt;"�@����U+%��N���u�C�:�-H�T�n��V�d��TT�Ai
��''�5 �T&lt;$�@��EEC�������r�R�,&gt;u�� ��s�%s%��(��M�=@�qe�&gt;�S����zl�_-��^eX+��]A��QQTK�]zh�X�~X�����*g��</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Jun 14 2019 11:24:56 GMT+0200 (czas środkowoeuropejski letni)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
<script src="../../../block-navigation.js"></script>
</body>
</html>
