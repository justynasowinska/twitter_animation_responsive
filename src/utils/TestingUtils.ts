import { ShallowWrapper, ReactWrapper } from 'enzyme';

/**
 * Gets Test ID for component for testing purposes.
 * According to React Native documentation, setting testID can make performance worse,
 * so we can set these ids only in development environment.
 * https://facebook.github.io/react-native/docs/view#testid
 * @param {string} testId  Test id that should be set.
 * @returns {string | undefined}    If is PROD envirenment these id will be undefined.
 */
export const setTestID = (testId: string): string | undefined => {
    if (__DEV__) {
        return testId;
    }

    return undefined;
};

/**
 * Get ReactWrapper or ShallowWrapper that contains found nodes.
 * @param {ShallowWrapper<any, any, any> | ReactWrapper<any, any, any>}  wrapper     Wrapper got by shallow or mount method.
 * @param {string}                                                       testId   Test Id which contains wrapper that we want to find.
 * @returns {ShallowWrapper<any, any, any> | ReactWrapper<any, any, any>} Returns ShallowWrapper (for shallow method) or ReactWrapper (for mount method).
 */
export const findByTestID = (
    wrapper: ShallowWrapper<any, any, any> | ReactWrapper<any, any, any>,
    testId: string
    ): ShallowWrapper<any, any, any> | ReactWrapper<any, any, any> => {
        return wrapper.findWhere((node: ShallowWrapper<any, any, any> | ReactWrapper<any, any, any>) => node.prop('testID') === testId);
};