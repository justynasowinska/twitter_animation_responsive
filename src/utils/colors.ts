export const colors = {
    headerBgColor: '#1DA1F2',
    usernameDarkTextColor: '#14171a',
    usernameLightTextColor: '#ffffff',
    avatarBorderColor: '#ffffff',
    appBackgroundColor: '#ffffff'
};