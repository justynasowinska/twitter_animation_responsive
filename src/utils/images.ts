export const images = {
    avatarImg: require('../../assets/images/me.jpg')
};