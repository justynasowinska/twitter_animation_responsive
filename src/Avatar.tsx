import React from 'react';
import { StyleSheet, Image, Animated, View, Text, ViewStyle } from 'react-native';
import { colors } from './utils/colors';
import { setTestID } from './utils/TestingUtils';

interface PropsType {
  image: any;
  username: string;
  containerStyles?: ViewStyle;
  animatedStyles: any;
  userNameFontSize: number;
}

export const Avatar = (props: PropsType) => {
    const { image, containerStyles, animatedStyles, username, userNameFontSize } = props;

    return (
        <View style={containerStyles}>
          <Animated.View style={[styles.imageContainer, animatedStyles]}>
              <Image
                  source={image}
                  style={styles.image}
              />
          </Animated.View>
          <View>
              <Text
                testID={setTestID('avatarUserName')}
                style={[styles.headerText,
                { fontSize: userNameFontSize }]}
              >
                {username}
              </Text>
          </View>
        </View>
    );
}

const styles = StyleSheet.create({
  imageContainer: {
    borderRadius: 100,
    borderColor: colors.avatarBorderColor,
    borderWidth: 3,
    overflow: 'hidden',
  },
  image: {
    flex: 1,
    width: undefined,
    height: undefined,
  },
  headerText: {
    fontWeight: 'bold',
    color: colors.usernameDarkTextColor
  },
});
