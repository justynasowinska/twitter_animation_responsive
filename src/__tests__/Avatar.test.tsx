import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import { Avatar } from '../Avatar';
import { findByTestID } from '../utils/TestingUtils';
import { images } from '../utils/images';

describe('Avatar component', () => {
    let component: ShallowWrapper;

    beforeEach(() => {
        component = shallow(
            <Avatar
                image={images.avatarImg}
                username="John_Doe"
                animatedStyles={undefined}
                userNameFontSize={15}
            />
        );
    });

    it('renders correctly', () => {
        expect(component).toMatchSnapshot();
    });

    it('renders correct username', () => {
        const username = 'Donald_Duck';
        component.setProps({ username });

        const foundElement = findByTestID(component, 'avatarUserName') as ShallowWrapper;

        expect(foundElement.dive().text()).toBe(username);
    });
});