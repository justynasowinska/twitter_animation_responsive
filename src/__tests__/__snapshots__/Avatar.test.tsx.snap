// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component renders correctly 1`] = `
<View>
  <AnimatedComponent
    style={
      Array [
        Object {
          "borderColor": "#ffffff",
          "borderRadius": 100,
          "borderWidth": 3,
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/images/me.jpeg",
        }
      }
      style={
        Object {
          "flex": 1,
          "height": undefined,
          "width": undefined,
        }
      }
    />
  </AnimatedComponent>
  <Text
    style={
      Array [
        Object {
          "color": "#14171a",
          "fontWeight": "bold",
        },
        Object {
          "fontSize": 15,
        },
      ]
    }
    testID="avatarUserName"
  >
    John_Doe
  </Text>
</View>
`;
